util/weekly_build/ShellSetup.sh

```diff
@@ -11,16 +11,21 @@ if [ ${RUNDIR::1} != "/" ]; then
   exit 1
 fi
 
-PACKAGES_TO_TEST=${PACKAGES_TO_TEST:-"libpng libaec jasper scotch w3emc g2 g2c"}
+# Defaults to empty, can be overwritten in site config
+PACKAGES_TO_TEST=""
 
 function alert_cmd {
   echo "Your run failed in $1. This is a placeholder alerting function. 'alert_cmd' should be defined for each system."
 }
 
-function spack_install_exe {
-  spack $* | tee -a log.install 2>&1
+function spack_wrapper {
```

Comment:
Introducing `spack_wrapper` that allows users to change how to pipe spack output into log files. Thinking a bit more, we may still want to keep spack_install_exe to support the special needs of running parallel installs ...